release_tester/arangodb/starter/deployments/selenium_deployments/__init__.py

```diff
@@ -61,7 +64,7 @@ def init(runner_type: RunnerType,
 
     if runner_type == RunnerType.DC2DCENDURANCE:
         from arangodb.starter.deployments.selenium_deployments.dc2dc_endurance import Dc2DcEndurance
-        return Dc2DcEndurance(*args)
+        return Dc2DcEndurance(driver)
```

Comment:
This seems unrelated. I cannot comment if this is correct change or not, as I am not familiar with the code.

The new version looks more consistent with other functionality, but I do not see counter part modification in Dc2DcEndurance.

Reply:
yes, this was an oversight pointed out by pylint. 

The endurance module isn't frequently used, hence it didn't suface yet.